prediction_market_agent/agents/safe_guard_agent/streamlit_pages/agents_page.py

```diff
@@ -22,7 +26,11 @@ def agents_page() -> None:
 
 Agent will also send you a message to your Safe with the result of the validation.
 """
-    )
+        )
+
+        agent_address = Config().SAFE_GUARD_AGENT_ADDRESS
+        st.markdown(
+            f"- `{agent_address}` - https://gnosisscan.io/address/{agent_address}"
+        )
```

Comment:
Should it be hardcoded to `gnosis scan` or is it chain-dependent?

Reply:
True! Fixed